### PR TITLE
Improving Tandem retry logic (UPLOAD-290)

### PR DESCRIPTION
**Pull request by gniezen**
Addresses:
- [UPLOAD-290]

[UPLOAD-290]: https://tidepool.atlassian.net/browse/UPLOAD-290

### CODE REVIEW

File: lib/drivers/tandem/tandemDriver.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ var debug = require('bows')('TandemDriver');
 var debugMode = require('../../../app/utils/debugMode');
 var common = require('../../commonFunctions');
 var uploadDataPeriod = require('../../../app/utils/uploadDataPeriod');
+var rollbar =require('../../../app/utils/rollbar');
```
Comment (pazaan) [inline lib/drivers/tandem/tandemDriver.js:32]: Nit:
```suggestion
var rollbar = require('../../../app/utils/rollbar');
```

File: lib/drivers/tandem/tandemDriver.js [hunk 2]
```diff
@@ -1133,28 +1134,67 @@ module.exports = function (config) {
...
+              }
+              else {
+                debug('Tandem found: ', result);
+
+                // cancel outstanding multi log request just in case
+                tandemCommand(COMMANDS.LOG_ENTRY_SEQ_MULTI_STOP_DUMP, null, function (err) {
+                  if(err) {
+                    callback(err,null);
+                  }
+                  else{
```
Comment (pazaan) [inline lib/drivers/tandem/tandemDriver.js:1178]: Nit:
```suggestion
                  } else {
```

**Comment by gniezen** [comment]
Removed test suffixes after ticket was approved. Ready for final check!

**Comment by gniezen** [comment]
Scratch that, it has not gone through QA yet, sorry!

**Comment by gniezen** [comment]
This has now been approved and is ready for final check.

**Pull request merged**